### PR TITLE
[DOCS] Draft VisualScript* descriptions

### PR DESCRIPTION
**Pull request by cbscribe**
Descriptions should include what data/sequence ports the node provides, so this is a first pass at outlining those. See the `<description>` section.

If we can arrive at a good standard for these three nodes, we can use them as guidelines for those who are documenting the remaining nodes.

### CODE REVIEW

File: doc/classes/VisualScriptSceneNode.xml [hunk 1]
```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptSceneNode" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Node reference.
 	</brief_description>
 	<description>
+		A direct reference to a node.
+		[b]Input Ports:[/b]
+        none
```
Comment (NathanLovato) [inline doc/classes/VisualScriptSceneNode.xml]: space indents on this line and the next 2

**Review by NathanLovato — COMMENTED** [review]
Looks good to me! @reduz got some suggestions to document the VisualScript classes in the reference?

File: doc/classes/VisualScriptOperator.xml [hunk 2]
```diff
@@ -4,6 +4,13 @@
 	</brief_description>
 	<description>
 	</description>
+	[b]Input Ports:[/b]
```
Comment (akien-mga) [inline doc/classes/VisualScriptOperator.xml]: Wrong indentation, and you missed the `<description>` tags :)

File: doc/classes/VisualScriptCondition.xml [hunk 3]
```diff
@@ -4,7 +4,14 @@
 		A Visual Script node which branches the flow.
 	</brief_description>
 	<description>
-		A Visual Script node which switches the flow based on a boolean. It acts similar to if/else in typical programming languages.
+		A Visual Script node that checks a [bool] input port. If [code]true[/code] it will exit via the “true” sequence port. If [code]false[/code] it will exit via the "false" sequence port. After exiting either, it exits via the “done” port. Sequence ports may be left disconnected.
+        [b]Input Ports:[/b]
```
Comment (akien-mga) [inline doc/classes/VisualScriptCondition.xml]: Wrong indentation.

**Comment by akien-mga** [comment]
Merged manually with fixed indentation and sections as  bf0b32819.

**Pull request closed without merging**